src/network/networkd-network.c

```diff
@@ -481,7 +481,13 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                               "IPv6Prefix\0"
                               "IPv6RoutePrefix\0"
                               "TrafficControlQueueingDiscipline\0"
-                              "CAN\0",
+                              "CAN\0"
+                              "ControlledDelay\0"
+                              "FairQueueTrafficPolicing\0"
+                              "FairQueueingControlledDelay\0"
+                              "NetworkEmulator\0"
+                              "StochasticFairnessQueueing\0"
+                              "TokenBufferFilter\0",
```

Comment:
the B in TBF stands for "bucket", not "buffer"

Reply:
Heh, that's a good point. Let's fix that, but in a separate patch, since this was a preexisting issue.